python/kwiver/vital/types/type_casters.h

```diff
@@ -0,0 +1,29 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#ifndef VITAL_TYPE_CASTER
+#define VTIAL_TYPE_CASTER
```

Comment:
I assume this should be VITAL_TYPE_CASTER?

Reply:
That would be correct....